src/wp-includes/media.php

```diff
@@ -5753,11 +5742,40 @@ function wp_get_loading_optimization_attributes( $tag_name, $attr, $context ) {
 	 * If the element is in the viewport (`true`), potentially add
 	 * `fetchpriority` with a value of "high". Otherwise, i.e. if the element
 	 * is not not in the viewport (`false`) or it is unknown (`null`), add
-	 * `loading` with a value of "lazy".
+	 * `loading` with a value of "lazy" and set `decoding` with the value of
+	 * "async".
 	 */
 	if ( $maybe_in_viewport ) {
 		$loading_attrs = wp_maybe_add_fetchpriority_high_attr( $loading_attrs, $tag_name, $attr );
 	} else {
+		// Only add `decoding` attribute if the feature is enabled.
+		if ( 'img' === $tag_name ) {
+			$decoding = array_key_exists( 'decoding', $attr ) ? $attr['decoding'] : 'async';
+
+			/**
+			 * Filters the `decoding` attribute value to add to an image. Default `async`.
+			 *
+			 * Returning a falsey value will omit the attribute.
+			 *
+			 * @since 6.1.0
+			 * @since 6.4.0 Added the `$attr` parameter
```

Comment:
```suggestion
			 * @since 6.4.0 Added the `$attr` parameter.
```

Add missing end period.